raiden/ui/cli.py

```diff
@@ -846,14 +859,14 @@ def _run_app():
                 kwargs['mapped_socket'] = mapped_socket
                 app_ = _run_app()
 
-        except socket.error as v:
-            if v.args[0] == errno.EADDRINUSE:
-                print(
-                    'ERROR: Address %s:%s is in use. '
-                    'Use --listen-address <host:port> to specify port to listen on.' %
-                    (listen_host, listen_port),
-                )
-                sys.exit(1)
+        except RaidenServicePortInUseError:
+            print(
+                'ERROR: Address %s:%s is in use. '
+                'Use --listen-address <host:port> to specify port to listen on.' %
+                (listen_host, listen_port),
+            )
+            sys.exit(1)
+        except socket.error:
             raise
```

Comment:
You can just remove this except clause since it's not doing anything with the `socket.error` except immediately re-raising it.